pages/portfolio.html

```diff
@@ -4,9 +4,44 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
+  <link href="../stylesheets/portfolio.css" rel="stylesheet">
+  <title>My Work</title>
 </head>
 <body>
-  
+  <header>
+    <h1>
+      My Work
+    </h1>
+    <nav>
+      <a href="./index.html">Home</a>
+      <a href="./about.html">About Me</a>
+    </nav>
+  </header>
+  <main>
+    <h2>
+      My most recent projects:
+    </h2>
+    <h3>
+      Task List API
+    </h3>
+    <p>
+      Used Flask framework with the SQLAlchemy library to build out an 
+      API for CRUD actions related to tasks and goals. Data stored in 
+      postgresql database.<br>
+      Deployed on <a href="https://task-list-api-mbuntin.herokuapp.com/tasks">
+      Heroku</a>. <br>
+      <a href="https://github.com/Morfiend/task-list-api">Github repo</a>
+    </p>
+    <h3>Swap Meet</h3>
+    <p>
+      Practiced OOP principles, helper functions, and test driven development 
+      in Python. <br>
+      <a href="https://github.com/Morfiend/swap-meet">Github repo</a>
+    </p>
```

Comment:
Nice! Another way to write the HTML for the portfolio page would be to have an unordered list and each list item would be a project. 

You could also put the h3 with the project name and p tag with the description in a section tag and then each LI tag could have the nested section